Added a js script for refreshing images

MrKomputerKid opened the pull request:
Images are in the relative path of images/ under the names image1.webp... etc

If you need to re-add the trailing slashes to the html and js, please do so before deploying. :)

I had to remove the trailing slashes from the image references in html and js probably due to the way windows handles files. But I dunno for sure, lol.

MrKomputerKid committed:
Added a js script for changing the background on page reload! Images available in /images.

nslee333 commented on background-change.js:
```diff
@@ -0,0 +1,25 @@
+window.addEventListener('load', function() {
+    // Array of background images in the images folder
+    const backgroundImageNames = [
+        'image1.webp',
```
This all looks great, the only thing I need changed is swapping the image names from `image1.webp` to`bg1.webp`, that way it's a little more obvious to someone outside the repo. Other than that looks great man!

MrKomputerKid committed:
renamed:    background-change.js -> background.js
renamed:    images/image1.webp -> images/bg1.webp
renamed:    images/image2.webp -> images/bg2.webp
renamed:    images/image3.webp -> images/bg3.webp
renamed:    images/image4.webp -> images/bg4.webp
renamed:    images/image5.webp -> images/bg5.webp
renamed:    images/image6.webp -> images/bg6.webp

Modified background.js to be compliant with new image names.

MrKomputerKid committed:
Updated index.html to reference background.js instead of background-change.js
